blog/2024-01-23/index.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: over-one-million-active-users-and-growing
+title: Over One Million Active Users—and Growing
+authors: [cassidyjames]
+tags: [flathub, milestone, stats]
+---
+
+Earlier this month we shared [new app metadata guidelines](../2024-01-08/index.md) in response to the growth and maturity of Flathub. Today we're proud to share about that growth in a bit more detail including a huge milestone, how we calculate stats, and what we believe is driving that growth.
+
+<!-- truncate -->
+
+## Milestones
+
+To date, Flathub has served a total of **1.6 billion downloads** of **over 2,400 apps** and their updates.
+
+Since introducing [verified apps](/docs/for-users/verification) last year, **over 800 apps have been verified** by their original authors—more than one third of all apps, with that number constantly increasing. 
+
+And finally, we're thrilled to announce that Flathub has surpassed **one million active users**. 🎉️
+
+## How We Measure Growth
+
+We have a [public dashboard](https://flathub.org/statistics) that shares some basic statistics, powered by an [open source script](https://github.com/flathub-infra/flathub-stats). We don't track individual metrics from end user machines; instead, we look at downloads of specific artifacts from the Flathub infrastructure itself. This gives us a count of installs and updates over time, downloads per country (based on the origin of requests), and some derivative data like downloads of apps in different categories.
+
+But how can we measure how many _active users_ we have?
+
+Since Flathub necessarily serves downloads of Flatpak _runtimes_ (common platforms on which apps are built), we can estimate active users fairly well. For example, when installing or updating most apps, Flatpak will automatically install the base [FreeDesktop SDK](https://freedesktop-sdk.gitlab.io/) runtime on which the KDE and GNOME runtimes (and most other apps) are built. We can look at the number of downloads of _updates_ to a recent release of this runtime to understand how many active installs are out there getting Flatpaks from Flathub.
```

Comment:
Not strictly true. Installing an app which uses `org.gnome.Platform` causes that runtime to be installed, but does not cause the `org.freedesktop.Platform` on which the GNOME runtime is based to be installed. Its extensions – maybe, but the link below is to the fdo runtime itself.

(I think using the freedesktop runtime as a proxy is probably fine; it's conservative but probably not too conservative.)

Reply:
LGT